CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [11.0.1] - 2022-07-06
```

Comment:
remove this version change since we don't know when this will be released. In general, your PRs should not have a version change associated with them. Also remove the edits from package.json and all